x-pack/plugins/security_solution/public/common/containers/events/last_event_time/index.ts

```diff
@@ -85,9 +85,6 @@ export const useTimelineLastEventTime = ({
                   lastSeen: response.lastSeen,
                   refetch: refetch.current,
                 }));
-              } else if (isErrorResponse(response)) {
-                setLoading(false);
-                addWarning(i18n.ERROR_LAST_EVENT_TIME);
```

Comment:
I think this translation's now unused; the linter will probably complain about that eventually.

Reply:
Unused translation removed 251df102e339dbf71469bb0ae89bff0ba41a82f4